### DISCUSSION TITLE
Quick Reblogs/Remember Reblogs Question

### DISCUSSION THREAD

**Discussion by jesperhc** [AprilSylph/XKit-Rewritten#1311] [Help Desk]
Does this feature also remember posts that you add to the queue before they're posted? If not, does it remember after they're posted? And am I correct in assuming that mobile reblogs aren't remembered? Sorry if the answer has been posted, but I couldn't find it.

**Comment by marcustyphoon** [comment]
Quick Reblog only remembers posts that you reblog using its popup (no mobile reblogs or reblogs with the full desktop post editor)! And yes, it remembers posts if you queue them with the popup (they're added to the list immediately when you press queue).
